projects/plugins/jetpack/modules/videopress/editor-media-view.php

```diff
@@ -14,15 +19,20 @@ function videopress_handle_editor_view_js() {
 
 	add_action( 'admin_print_footer_scripts', 'videopress_editor_view_js_templates' );
 
-	wp_enqueue_style( 'videopress-editor-ui', plugins_url( 'css/editor.css', __FILE__ ) );
+	wp_enqueue_style(
+		'videopress-editor-ui',
+		plugins_url( 'css/editor.css', __FILE__ ),
+		array(),
+		Jetpack_VideoPress::VERSION
```

Comment:
All this makes me wonder if we should move it all to `JETPACK__VERSION` to avoid all that complexity.

Reply:
I thought about that, but just left it with the VideoPress specific one for now, which I think will be sorted anyways if VideoPress moves standalone.